pkg/agent/route/route_linux.go

```diff
@@ -258,6 +258,17 @@ func (c *Client) syncIPTables() error {
 	var err error
 	v4Enabled := config.IsIPv4Enabled(c.nodeConfig, c.networkConfig.TrafficEncapMode)
 	v6Enabled := config.IsIPv6Enabled(c.nodeConfig, c.networkConfig.TrafficEncapMode)
+	proto := iptables.ProtocolDual
+	if !v4Enabled && v6Enabled {
+		proto = iptables.ProtocolIPv6
+	}
+	if v4Enabled && !v6Enabled {
+		proto = iptables.ProtocolIPv4
+	}
+	if !v4Enabled && !v6Enabled {
+		return fmt.Errorf("at least one of IPv4 and IPv6 should be enabled")
+	}
```

Comment:
If https://github.com/antrea-io/antrea/pull/2469/files#r680650520 is implemented, we don't need this `proto` check which is redundant with `v4Enabled` and `v6Enabled`, and can just call the methods with `ProtocolDual` in most cases.

Reply:
Removed.